tests/Fixtures/backtraces/anonymous_frame.json

```diff
@@ -3,13 +3,52 @@
     "line": 12,
     "backtrace": [
         {
-            "function": "triggerError",
-            "class": "TestClass"
+            "class": "class@anonymous/path/to/app/consumer.php0x7fc3bc369418",
```

Comment:
Please revert all the changes to this file and just add the single frame needed to reproduce the issue. All the other frames you added do not provide any useful information to the test this fixture is used in

Reply:
ok trimmed it down to just the one frame

Reply:
Sorry, what I meant was to leave the file exactly as it was and just add your frame. This is because each frame that used to be written here tested a specific case (like in a PHPUnit data provider)

Reply:
I agree with @ste93cry: we need two tests here, one with and one without memory addresses.

Reply:
ok fully restored the original test and copied mine to a new test

Reply:
Can you please name the JSON file `anonymous_frame_with_memory_address.json`?

Reply:
fixed